server_admin/topics/users/credentials.adoc

```diff
@@ -1,60 +1,28 @@
-[[_user-credentials]]
+[id="ref-user-credentials_{context}"]
 = User Credentials
 
-When viewing a user if you go to the `Credentials` tab you can manage a user's credentials.
+You can manage credentials of a user in the *Credentials* tab. 
 
 .Credential Management
 image:{project_images}/user-credentials.png[]
 
-The credentials are listed in a table, which has the following fields:
+The credentials are listed in a table, and contain the following fields:
 
 Position::
-   The arrow buttons in this column allows you to shift the priority of the credential for the user, with the topmost credential having the highest priority.
-   This priority determines which credential will be shown first to a user in case of a choice during login. The highest priority of those available to the
-   user will be the one selected.
+   The arrow buttons in the *Position* column allow you to shift the priority of the credential for the user. The topmost credential has the highest priority. The priority determines which credential is displayed first after a user logs in.
+
 Type::
-   This shows the type of the credential, for example `password` or `otp`.
+   This shows the type of the credential, for example *password* or *OTP*.
+
 User Label::
-   This is an assignable label to recognise the credential when presented as a selection option during login. It can be set to any value to describe the
-   credential.
-Data::
-   This shows the non-confidential technical information about the credential. It is originally hidden, but you can press `Show data...` to reveal it for a
+   This is an assignable label to recognize the credential when presented as a selection option during login. It can be set to any value to describe the
    credential.
-Actions::
```

Comment:
Was the Action Field moved else where?

Reply:
It's still in this module

Reply:
Okay, great.  Thanks for confirming.  I am sure it's in a "much better place now."